libraries/AP_HAL_VRBRAIN/GPIO.cpp

```diff
@@ -173,7 +173,9 @@ AP_HAL::DigitalSource* VRBRAINGPIO::channel(uint16_t n) {
 }
 
 /* Interrupt interface: */
-bool VRBRAINGPIO::attach_interrupt(uint8_t interrupt_num, AP_HAL::Proc p, uint8_t mode)
+bool VRBRAINGPIO::attach_interrupt(uint8_t interrupt_num,
+                                   irq_handler_fn_t fn,
+                                   INTERRUPT_TRIGGER_TYPE mode)
 {
     return true;
```

Comment:
What happened here? This should have the same as PX4 or we break VRBrain boards!